fxa-client/sdks/android/library/build.gradle

```diff
@@ -0,0 +1,109 @@
+apply plugin: 'com.android.library'
+apply plugin: 'org.mozilla.rust-android-gradle.rust-android'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.build['minSdkVersion']
+        targetSdkVersion rootProject.ext.build['targetSdkVersion']
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    // Help folks debugging by including symbols in our native libraries.  Yes, this makes the
+    // resulting AAR very large.  The Android ecosystem seems to be in flux around who is in charge
+    // of stripping native binaries, but for now let's provide symbols and see how consumers react.
+    packagingOptions {
+        doNotStrip "**/*.so"
+    }
+}
+
+cargo {
+    // The directory of the Cargo.toml to build.
+    module = '../../../../fxa-client/ffi'
+
+    // The Android NDK API level to target.
+    apiLevel = 21
+
+    // Where Cargo writes its outputs.
+    targetDirectory = '../../../../target'
+
+    libname = 'libfxa_client'
+
+    // The Cargo targets to invoke.  The mapping from short name to target
+    // triple is defined by the `rust-android-gradle` plugin.
+    targets = [
+        // 'default', // TODO: For unit tests.
+        'arm',
+        // 'arm64', // XXX: Just for development speed.
```

Comment:
need to uncomment this before merge?

Can gradle have a task that for each env, so in development you can pick the one you want and develop faster.

Reply:
Re: uncommenting -- this is in a DO NOT LAND commit.

Re: separate tasks, I have been thinking about how to achieve this.  I have Vague Ideas about how to do this by hooking into the Android-Gradle plugin, and will try to dig in ASAP.

Reply:
I'd love to see how this is done with gradle if so. I couldn't figure out how to do things conditionally in any way in the cargo block except for stuff in `exec`...

Reply:
Oh yeah, there's no uniform support for this at all.  If you look at the `rust-android-gradle` plugin, I did separate the `cargoBuild{Arm,Arm64,X86}` tasks; but they're not conditionally wired into the Android-Gradle plugin task tree.  That's non-trivial but clearly something that consumers (including me!) want, so I'll try to investigate it.